openff/system/tests/test_interop/test_openmm.py

```diff
@@ -21,32 +23,30 @@ def test_from_openmm_single_mols(mol, n_mols):
     mol = Molecule.from_smiles(mol)
     mol.generate_conformers(n_conformers=1)
     top = Topology.from_molecules(n_mols * [mol])
-    top.box_vectors = np.asarray([4, 4, 4]) * unit.nanometer
+    mol.conformers[0] -= np.min(mol.conformers) * unit.angstrom
+
+    if periodic:
+        top.box_vectors = np.eye(3) * np.asarray([10, 10, 10]) * unit.nanometer
+    else:
+        top.box_vectors = None
 
     if n_mols == 1:
         positions = mol.conformers[0]
     elif n_mols == 2:
         positions = np.vstack(
-            [mol.conformers[0], mol.conformers[0] + 2 * unit.nanometer]
+            [mol.conformers[0], mol.conformers[0] + 3 * unit.nanometer]
         )
+        positions = positions * unit.nanometer
```

Comment:
```suggestion
        positions = positions * unit.angstrom
```

I was also getting the nan energies with `periodic=True` and `n_mols=2`. I think there's an artificial inflation of positions when multiplying by `unit.nanometer`. When adding 3 nm to the conformer positions, this correctly adds 30 angstroms to the positions, but everything is still implicitly in angstroms, even when you convert everything to a numpy array. Then, multiplying by `unit.nanometer` to re-add units will incidentally increase the positions by an order of magnitude. I think it might be more appropriate to multiply by `unit.angstrom` (and this got my tests to pass locally)

packages:
```
openforcefield            0.8.2              pyh39e3cac_0    omnia
openforcefields           1.3.0                      py_0    omnia
openmm                    7.5.0            py37hd9ee695_4    conda-forge
```